src/pages/playlistSelector.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import TextField from "@material-ui/core/TextField";
+import MenuItem from "@material-ui/core/MenuItem";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    height: "100vh",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    backgroundImage: `url('./yas.png')`,
+    backgroundSize: "100% 100%",
+    margin: "-8px",
+    padding: "0",
+  },
+  select: {
+    width: "300px",
+  },
+}));
+
+function playlistSelectorPage() {
+  const playlists = [
+    { value: "default", label: "SELECT YOUR PLAYLIST" },
+    {
+      value: "Taylor Swift",
+      label: "Coven Girl music",
+    },
+    {
+      value: "Weeaboo",
+      label: "Naruto rawr xD",
+    },
+    {
+      value: "My Chemical Romance",
+      label: "Emo Dreams",
+    },
+    {
+      value: "TWICE",
+      label: "Just like TT",
+    },
+  ];
+  const classes = useStyles();
+  const [playlist, setPlaylist] = React.useState("default");
+  const handlePlaylist = (event: any) => {
+    setPlaylist(event.target.value);
+  };
+
+  return (
+    <body id="wrapper" className={classes.root}>
```

Comment:
can the body and div be combined or no

Reply:
bruhh good idea imma do it rn

Reply:
No it can't be combined actually because the div style is only for input but body is the whole screen and they have separate css